pvlib/irradiance.py

```diff
@@ -1369,6 +1374,7 @@ def disc(ghi, solar_zenith, datetime_or_doy, pressure=101325,
     am = atmosphere.get_relative_airmass(solar_zenith, model='kasten1966')
     am = atmosphere.get_absolute_airmass(am, pressure)
 
+    am = np.minimum(am, max_airmass)  # GH 450
```

Comment:
what's "GH 450" just curious

Reply:
Github issue 450. I've seen this convention in other python projects. In any case I should add something more useful and then maybe "see GH 450 for more"